api/src/main/java/com/epam/pipeline/manager/cluster/performancemonitoring/ResourceMonitoringManager.java

```diff
@@ -345,7 +345,7 @@ private void performNotify(PipelineRun run, double cpuUsageRate,
     }
 
     private void performStop(PipelineRun run, double cpuUsageRate) {
-        if (run.isNonPause() || isClusterRun(run)) {
+        if (run.isNonPause() || PipelineRunManager.isClusterRun(run)) {
```

Comment:
This method was changed in `develop` branch - please rebase

Reply:
Done